README.md

```diff
@@ -53,8 +53,16 @@ Set your editor to use spaces, not tabs.
 ### Lowercase all element and attribute names.
 
 
-<a name="for-empty-elements-do-not-use-xhtml-style-closing-syntax--—-just-use-"></a>
-### For "empty" elements, do not use XHTML-style closing syntax `<br />` — just use `<br>`.
+<a name="use-id-attributes-sparingly"></a>
+### Use ID attributes sparingly.
+
+* NEVER use IDs as CSS selectors; it makes style overrides unnecessarily difficult.
```

Comment:
Exception: This is occasionally required to override old CSS that we can't kill without a serious time commitment. (e.g. application.css in m3 is prime example)